tests/ctst/world/Zenko.ts

```diff
@@ -101,19 +182,22 @@ export default class Zenko extends World {
      * @param {object} result - contains both stderr and stdout from the CLI command.
      * @returns {boolean} - if the result is a success or a failure
      */
-    checkResult(result: any[]): boolean {
+    checkResult(result: Result | Utils.Command | Result[]): boolean {
+        let usedResult: Result | Utils.Command | Result[];
         if (!Array.isArray(result)) {
-            result = [result];
+            usedResult = [result] as Result[];
+        } else {
+            usedResult = result;
         }
         let decision = true;
-        result.forEach(res => {
+        usedResult.forEach(res => {
```

Comment:
It feels strange that a variable potentially not an array has a `forEach`, why not updating as follows:

```suggestion
        let usedResult: Result[] = Array.isArray(result) ? result : [result];
        let decision = true;
        usedResult.forEach(res => {
```

Reply:
Let's see what this does, I'll rebase the other PRs to see if anything breaks after all these changes

9103dbb8a3921d72239868a91c336abca0294aa2